components/user_settings/security/mfa_section/mfa_section.test.tsx

```diff
@@ -139,11 +157,17 @@ describe('MfaSection', () => {
         it('on error, should show error', async () => {
             const error = {message: 'An error occurred'};
 
-            const wrapper = mountWithIntl(<MfaSection {...baseProps}/>);
+            const wrapper: ShallowWrapper<any, any, MfaSection> = shallow(
+                <MfaSection {...baseProps}/>
+            );
+
+            const mockEvent = {
+                preventDefault: jest.fn()
+            } as unknown as React.MouseEvent<HTMLElement>;
```

Comment:
I do not like this, but I understand this is the only way to create a MouseEvent object without manually adding the many different properties it must have. Am I right?

Reply:
yes